src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -86,7 +86,7 @@ const ProfilePicture = ({
                   style={styles.profileImage(size, width)}
                   source={{
                     uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH,
-                    cache: 'cacheOnly'
+                    cache: 'web'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same issue as above. Changing the cache policy from 'cacheOnly' to 'web' can lead to increased data usage and slower loading times. Consider keeping the 'cacheOnly' policy or use a different approach to handle updates to the profile picture.
```diff
- cache: 'web'
+ cache: 'cacheOnly'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->